contentcuration/contentcuration/utils/channelcache.py

```diff
@@ -29,11 +29,11 @@ def get_all_channels(cls):
 
     @classmethod
     def get_public_channels(cls, *args, **kwargs):
-        return cache.get_or_set(
-            cls.PUBLIC_CHANNEL_CACHE_KEY,
-            lambda: Channel.get_public_channels(*args, **kwargs),
-            cls.PUBLIC_CHANNEL_CACHE_TIMEOUT
-        )
+        """
+        Get public channels, no longer cached as queries are lazy-loaded so at this point the query has not
+        been run.
+        """
+        return Channel.get_public_channels(*args, **kwargs)
```

Comment:
Hmm instead of modifying ChannelCache, shouldn't we just remove any calls to that for now? It's just meant as a wrapper for channel methods.

Reply:
aka change any calls to ChannelCache to just Channel

Reply:
Okay, if we go that route I think it'd be best to just remove ChannelCache for now. If we do revive it, it will probably be in some different form anyway.

Reply:
That's fine! It's best to approach some of the ideas from the Caching CoP before more heavy handed backend caching like this:

- downstream caching through headers
- async cache filling through something like `cacheback`